server/server.js

```diff
@@ -1,16 +1,15 @@
 import http from "node:http";
-
-import app from "./app";
+import app from "./app"; // Keeping this for future use
 import { connectDb, disconnectDb } from "./db";
 import config from "./utils/config";
 import logger from "./utils/logger";
 
 const server = http.createServer(app);
 
 server.on("listening", () => {
-	const addr = server.address();
-	const bind = typeof addr === "string" ? `pipe ${addr}` : `port ${addr.port}`;
-	logger.info("listening on: %s", bind);
+  const addr = server.address();
+  const bind = typeof addr === "string" ? `pipe ${addr}` : `port ${addr.port}`;
```

Comment:
In which case will `addr` be a string?

Reply:
I am not sure about this. This was there in the starter kit but I will look into it

Reply:
typeof addr === "string" checks if address is a string, which would indicate a Unix domain socket or named pipe.
If address is an object (i.e., it's bound to an IP address and port), the bind variable will be set to port ${addr.port}.
This way, the code correctly logs the binding information, whether it's a network address (port) or a Unix domain socket (pipe).

Reply:
Ok, you can leave it like this for now

Reply:
We never use a unix socket or pipe, so it will never be a string

Reply:
I have changed it